### DISCUSSION TITLE
Feature requests or plug-in requests

### DISCUSSION THREAD

**Discussion by CatTeaA** [PurpurMC/Purpur#1450] [__Dismissed__] [answered]
Is there any plugin that prohibits drop items, ships, and minecarts from entering the nether portal loading chunks, and only allow players and other entities to use the nether portal?
The main purpose of seeking this plugin is to disable the nether portal chunk loader
It would also be nice if Purpur introduced the ability to disable the Nether Portal Chunk Loader.

**Comment by CatTeaA** [comment]
purpur.yml:
entities-can-use-portals: false
only allowing players to use portals, destroying the bio farm, not what I want

**Reply by MetallicGoat** [reply]
If you have not found a solution yet, I could write you a small plugin to do that for you.

**Reply by CatTeaA** [reply]
Thank you very much, if you can launch this plugin, I believe there will be many people willing to use it, because I don't see the same type of plugin on the market

**Reply by MetallicGoat** [reply]
Ok I can look at it later today. My discord is "metallicgoat".

**Reply by MetallicGoat** [reply]
Done! If it works for you could you please mark my comment as the 'correct' answer? If you have any questions you can shoot me a dm on discord, "metallicgoat".

**Comment by MetallicGoat** [comment, marked as answer]
Here is the plugin. Sorry it took a while for me to get it to ya. Had to finish an assignment.

I threw it on my GitHub, you can download it from the releases tab.

https://github.com/MetallicGoat/PortalGaurd

**Reply by CatTeaA** [reply]
Thank you, I tested it, and it's true that dropped items can't activate the Nether portal. However, diffrent types of minecarts and diffrent types of boats can still trigger it. I can still create Nether portal loaders. Is it possible to allow only players and monsters to use the Nether portal? I've tried on Discord, but I couldn't get verified where I am. I'm sorry, and I really appreciate your willingness to help create the plugin.
🌸❤️❤️❤️❤️❤️🌸

**Reply by MetallicGoat** [reply]
I blocked boats and minecarts now

https://github.com/MetallicGoat/PortalGaurd/releases/tag/v1.0.1

**Reply by Huge-mistake** [reply]
Can you add "chest_boat" to the blacklist, I feel that it can be released on modrinth, the function is great!

**Reply by CatTeaA** [reply]
Great, I tested it in 1.20.1 and it worked perfectly.
Can prevent chest_boat from using the portal?
Thank you so much.❤️

**Reply by MetallicGoat** [reply]
I will add a config where you guys can configure what entity types you want blocked. Otherwise I will have to hard code every entity type, and that could be a pain for future minecraft updates

**Reply by CatTeaA** [reply]
Wow, that's configurable, that's great
👍

**Reply by MetallicGoat** [reply]
I will send a message here when I update it with the config later today. Probably wont have a chance for a couple hours though

**Reply by CatTeaA** [reply]
okay,really really appreciate your help❤️

**Reply by MetallicGoat** [reply]
Not gonna lie, the main reason why I am doing this is because I want this on my github account.

![image](https://github.com/PurpurMC/Purpur/assets/66706505/f753f126-6e53-42b6-a09b-379e82a37558)

https://github.com/Schweinepriester/github-profile-achievements

I still need one more awnser to get it though. 

This has been fun to work on too though

**Reply by MetallicGoat** [reply]
k I updated it to have a config, and added Chest_boat to it.

If you have any other small thing like this please lmk, cause I would like to get one more discussion question here answered

**Reply by CatTeaA** [reply]
Thanks again.😊
